keyserver/handler.go

```diff
@@ -0,0 +1,15 @@
+package keyserver
+
+import . "github.com/coniks-sys/coniks-go/protocol"
+
+func (server *ConiksServer) handleRegistration(reg *RegistrationRequest) (Response, error) {
+	if len(reg.Username) == 0 || len(reg.Key) == 0 {
```

Comment:
Yes, I still wonder if whether we want to perform some extra tasks in the handles in the future. Otherwise, it's completely redundant.